libvips/include/vips/meson.build

```diff
@@ -0,0 +1,169 @@
+public_other_headers = files(
+    'almostdeprecated.h',
+    'buf.h',
+    'connection.h',
+    'dbuf.h',
+    'debug.h',
+    'deprecated.h',
+    'dispatch.h',
+    'format.h',
+    'gate.h',
+    'generate.h',
+    'intl.h',
+    'mask.h',
+    'private.h',
+    'sbuf.h',
+    'semaphore.h',
+    'thread.h',
+    'threadpool.h',
+    'transform.h',
+    'util.h',
+    'vector.h',
+    'video.h',
+    'vips7compat.h',
+)
+
+public_headers = files(
+    'arithmetic.h',
+    'basic.h',
+    'colour.h',
+    'conversion.h',
+    'convolution.h',
+    'create.h',
+    'draw.h',
+    'error.h',
+    'foreign.h',
+    'freqfilt.h',
+    'header.h',
+    'histogram.h',
+    'image.h',
+    'interpolate.h',
+    'memory.h',
+    'morphology.h',
+    'mosaicing.h',
+    'object.h',
+    'operation.h',
+    'rect.h',
+    'region.h',
+    'resample.h',
+    'type.h',
+    'vips.h',
+)
+
+enumtypes = gnome.mkenums(
+    'enumtypes',
+    sources: files(
+        'resample.h',
+        'memory.h',
+        'create.h',
+        'foreign.h',
+        'arithmetic.h',
+        'conversion.h',
+        'util.h',
+        'image.h',
+        'colour.h',
+        'operation.h',
+        'convolution.h',
+        'morphology.h',
+        'draw.h',
+        'basic.h',
+        'object.h',
+        'region.h'
+    ),
+    h_template: 'enumtypes.h.in',
+    c_template: 'enumtypes.c.in',
+    install_header: true,
+    install_dir: 'vips'
```

Comment:
I think this needs to be:

```
    install_dir: 'include/vips'
```

Reply:
Well, `get_option('includedir') / 'vips'` anyway.